docs/production/upgrade-or-modify.md

```diff
@@ -207,7 +207,7 @@ instructions for other supported platforms.
    to back up the system:
 
     ```
-    supervisorctl stop all
+    /home/zulip/deployments/current/scripts/stop-server
```

Comment:
Hmm, I don't think we can use this command in these instructions, since 3.0 doesn't have this script yet?  

Might be better to leave this specific one as the old form.

Reply:
Mmm, reasonable.